.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
```

Comment:
@danacealfonso 
This file is not supposed to be pushed.

Reply:
fixed in 776fc8db9c272df767d20c16e6f8802a0c8f6128

Reply:
removed in d16615db003888a0ca5f55ab7d485851fcf20f9b